code/css/style.css

```diff
@@ -0,0 +1,198 @@
+/* style.css */
+
+/* setup */
+* {
+  box-sizing: border-box;
+}
+
+/* clear fix hack */
+.clearfix:after {
+  content: ".";
+  display: block;
+  clear: both;
+  line-height: 0;
+  height: 0;
+}
+
+.clear {
+  clear:both;
+}
+
+.alignleft {
+  float: left;
+}
+
+.alightright {
+  float: right;
+}
+
+/******************************************
+/* BASE STYLES
+/*******************************************/
+
+/* lyout */
+.container {
+  width: 960px;
+  margin: 0 auto;
+}
+
+body {
+  color: ;
+  background: #A5B8A7;
+  font-family: Amatic SC;
```

Comment:
If the font family you've loaded has a space in the name, it needs to be in quotes like "Amatic SC" to make sure it works across all browsers.